app/assets/stylesheets/hitobito/customizable/_wagon.scss

```diff
@@ -14,14 +14,14 @@
   font-size: 2.462em; // 32px
   line-height: 1.250em; // 40px
   @include single-text-shadow(white, 0, 1px, 0);
+  color: #038;
 
-  background: image-url("logo.png") transparent no-repeat 0px 0px;
+  background: image-url("logo.svg") transparent no-repeat 0px 0px;
```

Comment:
Ja, habe ich getestet und funktioniert IE9+. [Browser support auf caniuse.com](http://caniuse.com/#feat=svg-css)
Die PNGs habe ich in 8e1911e gefixt.